tests/foreman/performance/test_candlepin_concurrent_subscription_attach.py

```diff
@@ -0,0 +1,116 @@
+"""Test class for concurrent subscription by register and attach"""
+from datetime import date
+from robottelo.common import conf
+from robottelo.cli.subscription import Subscription
+
+from robottelo.performance.perf_concurrent_test import ConcurrentTestCase
+
+
+class ConcurrentSubAttachTestCase(ConcurrentTestCase):
+    """Concurrent Subscribe to Red Hat Satellite 6 Server by attach tests"""
+
+    @classmethod
+    def setUpClass(cls):
+        super(ConcurrentSubAttachTestCase, cls).setUpClass()
+
+        # parameters for concurrent register and attach test
+        cls.savepoint = conf.properties[
```

Comment:
You are defining cls.savepoint, cls.raw_file_name and cls.stat_file_name in every class that inherits ConcurrentTestCase. I think this setup can be moved to the base class. Also it can be rewritten to:

```python
cls.savepoint = conf.properties[
    'performance.test.savepoint2_enabled_repos']
today = date.today().strftime('%Y%m%d')
cls.raw_file_name = '{0}-{1}'.format(
           today,
           conf.properties['performance.csv.raw_att_file_name'],
)
cls.stat_file_name = '{0}-{1}'.format(
    today,
    conf.properties['performance.csv.stat_att_file_name'],
)